packages/styles/src/variables/components/_tag.scss

```
@@ -0,0 +1,20 @@
+@use './../color' as col;
+@use './../commons' as commons;
```

The `as` is not necessary, we usually only use an alias for the mixins, functions and placeholders but leave the variable namespaces as is.

```suggestion
@use './../commons';
```